provider/pkg/gen/python-templates/yaml/yaml.tmpl

```diff
@@ -3,17 +3,214 @@
+                opts.aliases = ["oldName"]
+
+
+        # Omit a resource from the Chart by transforming the specified resource definition to an empty List.
+        def omit_resource(obj, opts):
+            if obj["kind"] == "Pod" and obj["metadata"]["name"] == "test":
+                obj["apiVersion"] = "v1"
+                obj["kind"] = "List"
+
+
+        example = ConfigGroup(
+            "example",
+            files=["foo.yaml"],
+            transformations=[make_service_private, alias, omit_resource],
+        )
+        ```
+
+        :param str name: A name for a resource.
+        :param Optional[List[str]] files: Set of paths or a URLs that uniquely identify files.
+        :param Optional[List[str]] yaml: YAML text containing Kubernetes resource definitions.
+        :param Optional[pulumi.ResourceOptions] opts: A bag of optional settings that control a resource's behavior.
+        :param Optional[List[Tuple[Callable, Optional[pulumi.ResourceOptions]]]] transformations: A set of
+               transformations to apply to Kubernetes resource definitions before registering with engine.
+        :param Optional[str] resource_prefix: An optional prefix for the auto-generated resource names.
+               Example: A resource created with resource_prefix="foo" would produce a resource named "foo-resourceName".
+        """
+        if not name:
+            raise TypeError('Missing resource name argument (for URN creation)')
+        if not isinstance(name, str):
+            raise TypeError('Expected resource name to be a string')
+        if opts and not isinstance(opts, pulumi.ResourceOptions):
+            raise TypeError('Expected resource options to be a ResourceOptions instance')
+        if not files:
+            files = []
+        if not yaml:
+            yaml = []
+
+        __props__ = dict()
+
+        if resource_prefix:
+            name = f"{resource_prefix}-{name}"
+        super(ConfigGroup, self).__init__(
+            "kubernetes:yaml:ConfigGroup",
+            name,
+            __props__,
+            opts)
+
+        self.resources = pulumi.Output.from_input({})
+
+        _files: List[str] = []
+        for file in files:
+            if _is_url(file):
+                _files.append(file)
+            else:
+                _files += [f for f in glob(file)]
+
+        opts = pulumi.ResourceOptions.merge(opts, pulumi.ResourceOptions(parent=self))
+
+        for file in _files:
+            cf = ConfigFile(
+                file, file_id=file, transformations=transformations, resource_prefix=resource_prefix, opts=opts)
+            # Add any new ConfigFile resources to the ConfigGroup's resources
+            self.resources = pulumi.Output.all(cf.resources, self.resources).apply(lambda x: {**x[0], **x[1]})
+
+        for text in yaml:
+            # Rather than using the default provider for the following invoke call, use the version specified
+            # in package.json.
```

Comment:
reference to package.json here is irrelevant to python

Reply:
👍  I'll fix this in another PR